File: pset6/mario.py
import cs50

def main():
    pyramid()

def pyramid():
    while True:
        print("Please enter heigh between 1 and 23: ")
        height = cs50.get_int()
        if height > 0 and height < 24:
            break
    for i in range(height):
        for j in range(height + i + 2):
            if j < height - i - 1:
                print(" ", end="")
            elif j == height:
                print("  ", end="")
            else:
                print("#", end="")
        print("")
        
if __name__ == ("__main__"):
    main()

File: pset6/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do{
        printf("Please enter height between 1 and 23: \n");
        height = get_int();
    }while(height < 0 || height > 23);
    
    for(int i = 0 ; i < height ; i++)
    {
        for(int j = 0; j <= height + i + 1; j++)
        {
            if(j < height - i - 1)
                printf(" ");
            else if(j == height)
                printf("  ");
            else 
                printf("#");
        }
        printf("\n");
    }
    return 0;
}